packages/ExternalLink/src/ExternalLink.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import tokens from "@paprika/tokens";
+import NewTabIcon from "@paprika/icon/lib/NewTab";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import * as styled from "./ExternalLink.styles";
+
+const propTypes = {
+  /** Text for aria-label. */
+  ariaText: PropTypes.string,
+
+  /** Link text for showing. */
+  children: PropTypes.string,
+
+  /** Should have an underline, by default it's true */
+  hasUnderline: PropTypes.bool,
+
+  /** Link url for the target. */
+  href: PropTypes.string.isRequired,
+};
+
+const defaultProps = {
+  ariaText: null,
+  children: null,
+  hasUnderline: true,
```

Comment:
I'd prefer booleans with `false` by default. 
Here, for instance, `noUnderline: false` instead of `hasUnderline`. 

Compare: 

```js
<ExternalLink /> // underline by default
<ExternalLink hasUnderline /> // made no sense, since hasUnderline is true anyway
<ExternalLink hasUnderline={false} />
```
with
```js
<ExternalLink />
<ExternalLink noUnderline />
```

@nahumzs @alexzherdev @mikrotron

Reply:
Personally i agree with that. It was api from older component so i left it unchanged, but perhaps now is good time. Your suggestion makes more sense to me...

Reply:
As a counter, we like to prefix our boolean props with `is`, `has`, `should`, etc... and `hasNoUnderline` is a bit meh... I'm not a big fan of _negative_ booleans, typically.  So while _in general_ I agree that defaults should be `false`, I'm okay with exceptions too. 

Tristan is right though, now is the right time to revisit the API and make any changes we like.  I'm personally 50/50 on this one, so I'll let Tristan make the call.

Reply:
IMO @AndreyChernykh 's point overweighs the naming concern (especially if we don't expect this prop to be used much anyway)

Reply:
Ok I'm going with `hasNoUnderline` with default value of `false` for now. I think having a default value of true doesn't make sense to me. 
Incidentally we have a prop `hasUnderline` on the `Heading` component as well, but its default value is `false` so I don't think we should change that.